lib/Serialization/ModuleFile.cpp

```diff
@@ -2531,6 +2531,11 @@ void ModuleFile::lookupObjCMethods(
   }
 }
 
+void ModuleFile::lookupImportedSPIs(const ModuleDecl *importedModule,
+                                    SmallVectorImpl<Identifier> &spis) const {
+  // TODO
```

Comment:
Probably this won't be needed at all, so maybe lookupImportedSPIs should be a method on SourceFile only?

Reply:
We need this one when printing the private textual interface at the merge-module phase as it needs all the SPI imported from all partial modules.